src/arfs_entity_result_factory.ts

```diff
@@ -1,5 +1,6 @@
 import { ArFSFileMetadataTransactionData } from './arfs_trx_data_types';
-import { TransactionID, DriveID, FolderID, FileID, FileKey, DriveKey } from './types';
+import { DriveID, FolderID, FileID, FileKey, DriveKey } from './types';
+import { TransactionID } from './types/transaction_id';
```

Comment:
Can't we export the safe types from `./types/index.ts`?

Reply:
3824924
Exported as a folder.

Reply:
It looks better! **-** We are having conflicts importing between `src/types.ts` and `src/types/`. Looking forward to that change

Reply:
Maybe I'd expect that change at #130 or further PRs for PE-635

Reply:
Yea I'll take care of it for ByteCount